src/components/task/repo/task.repo.js

```diff
@@ -0,0 +1,133 @@
+const _ = require('lodash');
```

Comment:
I am not sure, if we need this helper library here, just to check that an object is (not) empty. But since this library is part of our project, I think it can be used.